grammar/TypeQL.g4

```diff
@@ -57,7 +57,7 @@ query_match_group_agg :   query_match   match_group       match_aggregate  ;
 
 // MATCH QUERY MODIFIERS =======================================================
 
-modifiers             : ( filter';' )? ( sort';' )? ( offset';' )? ( limit';' )?;
+modifiers             : ( filter ';' )? ( sort ';' )? ( offset ';' )? ( limit ';' )?;
```

Comment:
@haikalpribadi could you confirm this doesn't change grammar semantics? I had to do it to simplify the adapter implementation (to have a simpler regex - keyword between two whitespaces)